src/xdocs/config_filters.xml

```diff
@@ -986,5 +986,277 @@ public class UserService {
+                      <td><code>null</code></td>
+                      <td>8.5</td>
+                  </tr>
+              </table>
+              <p>
+                  offCommentFormat and onCommentFormat must have equal <a
+                      href="https://docs.oracle.com/javase/7/docs/api/java/util/regex/Matcher.html#groupCount()">paren counts</a>.
+              </p>
+          </subsection>
+          <subsection name="Examples">
+              <p>
+                  To configure a filter to suppress audit events between a comment
+                  containing <code>CHECKSTYLE:OFF</code> and a comment containing
+                  <code>CHECKSTYLE:ON</code>:
+              </p>
+              <source>
+                  &lt;module name=&quot;Checker&quot;&gt;
+                  ...
+                  &lt;module name=&quot;SuppressWithPlainTextCommentFilter&quot;/&gt;
+                  ...
+                  &lt;/module&gt;
+              </source>
+              <p>
+                  To configure a filter to suppress audit events between a comment
+                  containing line <code>BEGIN GENERATED CODE</code> and a comment
+                  containing line <code>END GENERATED CODE</code>:
+              </p>
+              <source>
+                  &lt;module name=&quot;Checker&quot;&gt;
+                  &lt;module name=&quot;SuppressWithPlainTextCommentFilter&quot;&gt;
+                  &lt;property name=&quot;offCommentFormat&quot; value=&quot;BEGIN GENERATED CODE&quot;/&gt;
+                  &lt;property name=&quot;onCommentFormat&quot; value=&quot;END GENERATED CODE&quot;/&gt;
+                  &lt;/module>
+                  &lt;/module&gt;
+              </source>
+              <source>
+                  //BEGIN GENERATED CODE
+                  my.property=value1 // No violation events will be reported
+                  my.property=value2 // No violation events will be reported
+                  //END GENERATED CODE
+                  . . .
+              </source>
+              <p>
+                  To configure a filter so that <code>// stop tab
+                  check</code> and <code>// resume tab check</code> marks allowed tab positions:
+              </p>
+              <source>
+                  &lt;module name=&quot;Checker&quot;&gt;
+                  &lt;module name=&quot;SuppressWithPlainTextCommentFilter&quot;&gt;
+                  &lt;property name=&quot;offCommentFormat&quot; value=&quot;stop tab check&quot;/&gt;
+                  &lt;property name=&quot;onCommentFormat&quot; value=&quot;resume tab check&quot;/&gt;
+                  &lt;property name=&quot;checkFormat&quot; value=&quot;FileTabCharacterCheck&quot;/&gt;
+                  &lt;/module>
+                  &lt;/module>
+              </source>
+              <source>
+                  //stop tab check
+                  public static final int someConstant; // won't warn here if there is a tab character on line
```

Comment:
Previous comment at https://github.com/checkstyle/checkstyle/pull/5213#discussion_r152163093 .
There are still a few Java examples.

Reply:
@rnveach 
User is still able to use the filter to filter AuditEvents from FileSetChecks. FileSetChecks can see the java code as plain text too. The author of the issue has problem running checkstyle against java code.

Reply:
Yes, but only one example is not Java. I count 5 that are Java. I thought we would lean more to non-Java examples since this is made more for non-Java files and SuppressionCommentFilter should be for Java.

Reply:
Done. Add non-java examples. 
See https://mezk.github.io/i4841-SuppressWithPlainTextCommentFilter/site/config_filters.html#SuppressWithPlainTextCommentFilter